app/models/reports/bulk_recommend_empty_report.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Reports
+  class BulkRecommendEmptyReport < TemplateClassCsv
+    # required headers
+    TRN = "TRN"
+    TRAINEE_ID = "Provider trainee ID"
+
+    IDENTIFIERS = [TRN, TRAINEE_ID].freeze
+    DATE        = "Date QTS or EYTS standards met"
```

Comment:
I think we should probably let Rubocop enforce these sort of alignments if we want them (personally I don't but not sure on everyone else's taste)

Reply:
Changed to more natural layout. I'm not a fan of this kind of alignment either, I think it was copied from elsewhere TBH.